tests/stdio-test.cc

```diff
@@ -1,4 +1,5 @@
 #include <cstdio>
+#include <cstring>
```

Comment:
My editor appears to have added this include automatically, presumably because of using strcmp. Doesn't appear to be required but also not doing any harm?